apps/api/src/modules/tenant/tenant.service.test.ts

```
@@ -0,0 +1,227 @@
+import { beforeEach, describe, expect, it, type Mock, vi } from 'vitest'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)